core/src/main/java/com/linecorp/armeria/common/stream/StreamMessageWriter.java

```diff
@@ -20,5 +20,5 @@
  * A type which is both a {@link StreamMessage} and a {@link StreamWriter}. This type is mainly used by tests
  * which need to exercise both functionality.
```

Comment:
Could you also update the description?

Reply:
✅ updated based on your suggestion. can you check it?